src/lib/DAC/DAC.cpp

```diff
@@ -1,10 +1,14 @@
 
-#ifdef TARGET_R9M_TX
+#if defined(TARGET_R9M_TX) || defined(TARGET_TX_ES915TX)
```

Comment:
Would it better to define
`GPIO_PIN_DAC_SCL` and `GPIO_PIN_DAC_SDA` 
Then use those here to make this a bit generic? Maybe move those arrays into target.h (define)?

Reply:
Probably, but that's more work than I'm willing to do in this PR...

Reply:
ok, let's do it later.